run.go

```diff
@@ -153,6 +158,25 @@ func runAction(clicontext *cli.Context) error {
 		return errors.Errorf("unknown network %q", netstr)
 	}
 
+	securityOpt := clicontext.String("security-opt")
```

Comment:
`clicontext.StringSlice`

Reply:
I've been thinking about that, but ` --security-opt "seccomp=hoge.json,apparmor=foo.profile"`

Reply:
Docker doesn't support such syntax either

Reply:
I supposed that docker are the docker seems to be able to handle the parser in the same way.

Reply:
>Docker doesn't support such syntax either
Oh, I see.

Reply:
`docker run --security-opt seccomp=hoge.json --security-opt apparmor=foo.profile" is the correct form